src/app.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="%paraglide.lang%">
+<html lang="%paraglide.lang%" data-theme="">
```

Comment:
Instead of replacing the string `data-theme=""`, consider replacing something like `%theme%` instead. Although it may be unlikely, `data-theme=""` could be used in other instances (as a somewhat related example, it's used for hard-coding the theme for nollning pages).

```suggestion
<html lang="%paraglide.lang%" data-theme="%theme%">
```